tests/integration/targets/azure_rm_openshiftmanagedcluster/tasks/main.yml

```diff
@@ -106,6 +106,47 @@
 - assert:
       that: output['clusters'] | length >= 1
 
+- name: Fetch kubeconfig file to register
+  azure_rm_openshiftmanagedclusterkubeconfig_info:
+    name: "{{ cluster_name }}"
+    resource_group: "{{ resource_group }}"
+  register: kubeconfig_reg
+
+- assert:
+      that: kubeconfig_reg['kubeconfig'] | length >= 1
+
+- name: Fetch kubeconfig file with specific name
+  azure_rm_openshiftmanagedclusterkubeconfig_info:
+    name: "{{ cluster_name }}"
+    resource_group: "{{ resource_group }}"
+    path: "/home/testuser/{{ cluster_name }}"
+
+- name: Check for created named kubeconfig file
+  stat:
+    path: "/home/testuser/{{ cluster_name }}"
+  register: named_kubeconf_file
+
+- assert:
```

Comment:
Use `ansible.builtin.assert` instead of ‘assert' and all tasks should be named.

Reply:
Done